lcplants.py

```diff
@@ -32,24 +35,18 @@ def dicts(t): return {k: dicts(t[k]) for k in t}
 #pp.pprint(dicts(taxons()))
 
 
-
-
-def readcsv(): #runs, but utf-16 characters come through this one as /u00 code
-      with open('USDAsearch.txt', encoding = 'utf-8', errors = 'ignore') as csvfile:
-          reader = csv.DictReader(csvfile, delimiter=',', quotechar='"')
-          return [x for x in reader]
       
-def ioreadcsv(): #trips "Error: field larger than field limit (131072)"
-      with io.open('USDAsearch.txt', 'r', encoding = 'utf-16-le') as csvfile:
```

Comment:
Your indentation was weird here for some reason. I didn't think Python would run code like this, but it seems to do it!